src/csharp/examples/server/API/Register/RegisterController.cs

```diff
@@ -0,0 +1,61 @@
+using Microsoft.AspNetCore.Mvc;
+using System;
+using SessionlessExample.Server.Models;
+using System.Text.Json;
+using SessionlessNET.Impl;
+using SessionlessNET.Models;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SessionlessExample.Server.Controllers
+
+{
+    [ApiController]
+    [Route("/register")]
+    public class RegisterController: ControllerBase
+    {
+        Vault Vault;
+        Sessionless Sessionless;
```

Comment:
both Vault and Sessionless are not safely set as nullable or safely initialized

C# is smart enough to detect when the value could not possibly be null so feel free to make the fields nullable then keep your existing code which initializes them within the function scope, and c# wont have a problem with them since you clearly initialized them, then you will NOT need to call with null in mind (`thing?.child` -> `thing.child`)